sdk/core/Azure.Core/src/Pipeline/Internal/ResponseBodyPolicy.cs

```diff
@@ -15,6 +15,7 @@ namespace Azure.Core.Pipeline
     /// </summary>
     internal class ResponseBodyPolicy : HttpPipelinePolicy
     {
+        private const string NetworkTimeoutOverrideProperty = "NetworkTimeoutOverride";
```

Comment:
Should we actually expose this somehow (common headers?) to avoid typos, improve discoverability, etc?

Reply:
Or maybe expose some kind of static method like `SetNetworkTimeoutOverride` so that clients don't need to mess with message properties directly.

Reply:
I was thinking about it. I'm not even sure on which type would the constant go. My other option was adding a strongly typed property on HttpMessage as we have for buffering.

Reply:
Looking at `HttpMessage`, that's a good idea. I wonder, though: if we utilize this facility more and more, should we maybe use extension methods to decrease "bloat" on projects that don't need it? This alone or even ` bool BufferResponse` aren't enough, but it does set a precedent for a public type.

Reply:
I'll keep it as a property for now. If we ever want to add a strongly typed variable we would implement it with a fallback to the property.